tests/Functional/Admin/SharedBlockAdminTest.php

```diff
@@ -82,18 +82,18 @@ public function testFormsUrls(string $url, array $parameters, string $button, ar
     public static function provideFormUrlsCases(): iterable
     {
         // Blocks From SonataBlockBundle
-        // yield 'Create Shared Block - Container' => ['/admin/tests/app/sonatapageblock/shared/create', [
-        //     'uniqid' => 'shared_block',
-        //     'type' => 'sonata.block.service.container',
-        // ], 'btn_create_and_list', [
-        //     'shared_block[name]' => 'Name',
-        //     'shared_block[enabled]' => 1,
-        //     'shared_block[settings][code]' => 'code',
-        //     'shared_block[settings][layout]' => '{{ CONTENT }}',
-        //     'shared_block[settings][class]' => 'custom_class',
-        //     'shared_block[settings][template]' => '@SonataPage/Block/block_container.html.twig',
-        //     'shared_block[children]' => '',
-        // ]];
+        yield 'Create Shared Block - Container' => ['/admin/tests/app/sonatapageblock/shared/create', [
+            'uniqid' => 'shared_block',
+            'type' => 'sonata.block.service.container',
+        ], 'btn_create_and_list', [
+            'shared_block[name]' => 'Name',
+            'shared_block[enabled]' => 1,
+            'shared_block[settings][code]' => 'code',
+            'shared_block[settings][layout]' => '{{ CONTENT }}',
+            'shared_block[settings][class]' => 'custom_class',
+            'shared_block[settings][template]' => '@SonataPage/Block/block_container.html.twig',
+            // 'shared_block[children]' => '',
```

Comment:
https://github.com/sonata-project/SonataBlockBundle/blob/4.x/src/Block/Service/ContainerBlockService.php#L64

Not sure if that works. I think yes, but I don't know how to test it yet.